pkgs/applications/misc/kratos/default.nix

```
@@ -1,8 +1,16 @@
-{ fetchFromGitHub, buildGoModule, lib, stdenv }:
-
-buildGoModule rec {
+{
+  fetchFromGitHub,
+  buildGoModule,
+  lib,
+  stdenv
+}:
+let
   pname = "kratos";
   version = "1.1.0";
+  commit = "f47675b82012e0ff74b05b9b7e713b3aa2fdda54";
+in
+buildGoModule rec {
```

```suggestion
buildGoModule {
```
doesn't look like this is needed anymore

---

Thanks!